split-webapp/split/src/pages/Login.js

```diff
@@ -23,6 +23,16 @@ class Login extends Component {
     };
   }
 
+  componentDidMount = () => {
+    const inputField = document.getElementById("inner-container-id");
+    inputField.addEventListener("keyup", function(event) {
+      if (event.keyCode === 13) {
```

Comment:
Looks like `keyCode` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode). Would using [`event.key === "enter"`](https://caniuse.com/#feat=keyboardevent-key) be a safer choice?

Reply:
Changes made